### DISCUSSION TITLE
PathLayer Overlay with google maps only - sorry!

### DISCUSSION THREAD

**Discussion by nashworks** [visgl/deck.gl#7336] [Q&A]
can you recommend anything to get rid of overlap of 2 layers like that in the pic - 
they are on top of each other with no layers between them - 
![e552e440-566d-484a-b82a-b68bbb16eb35](https://user-images.githubusercontent.com/8750161/196146037-8e986a52-78f1-4a84-bd5c-562eed1d9bc0.png)

**Comment by nashworks** [comment]
by overlap, I mean the artifacts that bleed from second layer below into first layer above -

**Comment by nashworks** [comment]
this is version 8.8 - also this only happens when in tilted map view - when in northup view all seems to be fine - as you can see what i am attempting is to create 2 layers one on top of another - to get top layer path and bottom layer as a stroke/border for the top layer - maybe there is a different way to achieve bordered path?

**Reply by AdriSolid** [reply]
[This codepen](https://codepen.io/Pessimistress/pen/MWyaWxV) can help.

Also adding to the layer:
```javascript
  parameters: {
    depthTest: false,
  }
```

**Comment by nashworks** [comment]
yup - tried that already but it does not change the overlapping issue as you can see from the screenshot in post#1 above - 

appreciated for the code pan but it has same issue with overlapping artifacts and jagged edges 100% everywhere you look at it in the tilted map - 

![e4996b34-1e1a-4153-a4cc-f96d634427ba](https://user-images.githubusercontent.com/8750161/196165229-f99b6921-6b81-4f16-acde-af035b3f8689.png)

**Reply by AdriSolid** [reply]
Not sure if you tried [here](https://codepen.io/Pessimistress/pen/MWyaWxV):
```javascript
new StrokedPathLayer({
  data: [POINTS],
  getPath: d => d,
  getWidth: 2000,
  getColor: [255, 0, 0],
  getOutlineWidth: 6000,
  getOutlineColor: [255, 200, 20],
  parameters: {
    depthTest: false
  }
})
```

**Comment by nashworks** [comment]
**did you try to tilt** using 2 fingers gesture on your mobile screen or ALT button with click and hold mouse on your pc screen - if try that first then you will see your example has same issue with overlapping artifacts and jagged edges 100% everywhere you look at it in the tilted perspective view  - https://user-images.githubusercontent.com/8750161/196165229-f99b6921-6b81-4f16-acde-af035b3f8689.png

**Reply by AdriSolid** [reply]
Related with your browser?
https://user-images.githubusercontent.com/25487037/196366809-21561999-c7d0-4167-a3d8-9dc2fa826903.mp4

**Comment by nashworks** [comment]
ah wow! so you can see differently than me - how?? is it even possible?? i use latest chrome on intel7 and top of the line nvidia card - i am completely perplexed about the whole ordeal with deck.gl script - google maps have no problem with path routes and polylines except when you customize them, they overlap the street names - but deck.gl goes under the street names - that is why i saw that purdy amazing to utilize until you point me how it shows on your end when in fact you saw how horrible it shows on my end - so if it cannot match google maps polylines quality - why even bother as i cant go around telling users to get the right browser and computer and video card - and i am amazed of something else - no one from deck.gl team can reply - that is a total eff up in my humble but honest opinion - cheers! and thanks for a seat in the last row at a movie theater -

**Comment by nashworks** [comment]
just found something related: 

https://github.com/visgl/deck.gl/discussions/6168
